bin/si-smoketest/main.ts

```diff
@@ -1,90 +1,61 @@
 // main.ts
 import assert from "node:assert";
 import { SdfApiClient } from "./sdf_api_client.ts";
+import { createDefaultTestReportEntry, printTestReport, TestReportEntry } from "./test_execution_lib.ts";
+import { parseArgs } from "./binary_exeuction_lib.ts"; 
 
 if (import.meta.main) {
-  assert(Deno.env.get("SDF_API_URL")?.length > 0, "Expected SDF_API_URL env var");
-  assert(Deno.env.get("AUTH_API_URL")?.length > 0, "Expected AUTH_API_URL env var");
```

Comment:
Was the intent to remove this checks? I can see them anywhere else

Reply:
Ah good shout, this was indeed a mistake during pulling the arg parsing out, will fix

Reply:
Amended, sorry about that! I did have the feeling I missed something, see:
`checkEnvironmentVariables(Deno.env.toObject());`